CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Changed
 - Validate max. 1000 commands are added to `campaign()`, `events()`, `setupItemProperties()` and `deleteItemProperties()` requests (in accordance with Matej batch API limit).
+- `Lmc\Matej\Model\Response` now contains `getCommandResponse(int $index)` to access individual responses
+- `sorting()->send()` request now returns `Matej\Model\Response\SortingResponse` instance
```

Comment:
"`sorting()->send()` request now returns `SortingResponse` instance which provides shortcut methods." or something like this?